addons/stock/models/stock_move.py

```diff
@@ -248,7 +248,6 @@ def _compute_show_details_visible(self):
             else:
                 move.show_details_visible = (((consignment_enabled and move.picking_code != 'incoming') or
                                              show_details_visible or move.has_tracking != 'none') and
-                                             move.state != 'draft' and
```

Comment:
maybe too early to review, but draft MOs might need to be checked that the button doesn't show when we don't want it to :wink: